tools/ota_e2e_tests/aws_ota_test/aws_ota_test_runner.py

```diff
@@ -143,5 +143,11 @@ def runTests(self):
     def __cleanup(self):
         """Free resources used by the OTA agent and the flash serial object.
         """
-        self._otaAwsAgent.cleanup()
-        self._flashComm.cleanup()
\ No newline at end of file
+        # We will want to close the serial flash thread first so that there are no outstanding threads
+        # in case cleaning AWS Resources causes exceptions.
+        self._flashComm.cleanup()
```

Comment:
What if the flashcom.cleanup thrwos? How about?

```
try:
    self._flashComm.cleanup()
 except Exception as e:
    print(e)

try:
    self._otaAwsAgent.cleanup()
 except Exception as e:
    print(e)
```

Reply:
I knew someone was going to ask about this. It hasn't happened yet so I figured I would leave it. I just will update.